chat-client/src/ui/apps/amazonqCommonsConnector.ts

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ChatItemAction } from '@aws/mynah-ui'
+import { ExtensionMessage } from '../commands'
+import { AuthFollowUpType } from '../followUps/generator'
+
+export type WelcomeFollowupType = 'continue-to-chat'
+
+export interface ConnectorProps {
+    sendMessageToExtension: (message: ExtensionMessage) => void
```

Comment:
Missed refactoring to `sendMessageToClient`?

Reply:
changed it